workable-module-connector/0.1/modules/ROOT/pages/workable-module-connector-reference.adoc

```diff
@@ -0,0 +1,862 @@
+= Workable Module Connector 0.1.0-BETA Reference
+
+== Configurations
+[[Config]]
+=== Config
+
+Default configuration.
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<Config_BearerAuth, Bearer Auth>>
+ | The connection types to provide to this configuration. | | x
+| Name a| String | The identifier of this element used to reference it in other components. |  | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime will actually purge the instances when it sees it fit. |  | 
+|===
+
+==== Connection Types
+[[Config_BearerAuth]]
+===== Bearer Auth
+
+Provides access to the Workable Module API using a bearer token.
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| baseUri a| String | The base URI for the Workable Module API. |  | x
+| token a| String | The API key token for authentication. |  | x
+| TLS Configuration a| <<Tls>> | The TLS configuration for secure connections. |  | 
+| Reconnection a| <<Reconnection>> | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+
+== Supported Operations
+* <<AddCandidateComment>>
+* <<AddCandidateRating>>
+* <<CopyCandidateToJob>>
+* <<CreateCandidateForJob>>
+* <<CreateCandidateInTalentPool>>
+* <<DisqualifyCandidate>>
+* <<GetAllCandidates>>
+* <<GetCandidateActivities>>
+* <<GetCandidateById>>
+* <<GetCandidateOffer>>
+* <<MoveCandidateToStage>>
+* <<RevertCandidateDisqualification>>
+* <<UpdateCandidate>>
+* <<UpdateCandidateTags>>
+
+== Operations
+
+[[AddCandidateComment]]
+=== addCandidateComment
```

Comment:
The header leveling seems to be wrong in this reference. All the operations should be H2 and, Parameters/Output/For Configurations/Throws as H3. Refer to your PagerDuty reference as an example

https://github.com/mulesoft/docs-connectors/pull/3044/files